packages/uicore/src/components/MultiStepProgressIndicator/MultiStepProgressIndicator.tsx

```diff
@@ -51,49 +54,57 @@ const Dot: React.FC<{ status: StepStatus; name?: string }> = ({ status, name })
   }
 }
 
-const Bar: React.FC<{ status: StepStatus }> = ({ status }) => {
+const Bar: React.FC<{ status: StepStatus; barWidth?: number }> = ({ status, barWidth }) => {
+  const fullBarWidth = barWidth ? { width: `${2 * barWidth}px` } : {}
```

Comment:
This is misleading right? **barWidth** is passed as a prop and we using twice the width

Reply:
Nvm, component classes need to be updated to make this clearer